content/en/docs/ibm/install-kubeflow.md

```diff
@@ -36,37 +36,46 @@ environment variables.
     ```
     export CLUSTER_NAME=kubeflow
     export CLUSTER_ZONE=dal13
+    export WORKER_PROVIDER=classic
     ```
 
     - `CLUSTER_NAME` must be lowercase and unique among any other Kubernetes
       clusters in the specified CLUSTER_ZONE.
-    - `CLUSTER_ZONE` identifies the location where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center. 
+    - `CLUSTER_ZONE` identifies the location where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center.
+    - `WORKDER_PROVIDER` represents the IBM cloud Kubernetes cluster with worker nodes on classic infrastructure.
+
+**Notices**:
+* It uses the worker node provider `classic` in this guide because it's the only one that supports worker nodes with GPUs.
```

Comment:
```suggestion
* It uses the worker nodes provider `classic` in this guide because it's the only one that supports worker nodes with GPUs.
```

Reply:
addressed in https://github.com/kubeflow/website/pull/2028/commits/9c471a4aecf999fdeb136ae19e68240ba9b290cb